src/components/Header.tsx

```diff
@@ -18,6 +22,18 @@ const Header = () => {
                         </picture>
                     </a>
                 </div>
+                <div className="contacts">
+                    <a href="/" className={`${currentPath === '' ? 'line' : ''}`}>
```

Comment:
I'd suggest considering the usage of 'classnames' library. It is quite popular and works perfectly for situations like yours when you need to add a class or nothing

Reply:
thanks for the advice, will update the request with usage of this library.

Reply:
https://reactrouter.com/web/api/NavLink/activeclassname-string